test/models/report_test.rb

```diff
@@ -49,7 +49,7 @@ class ReportTest < ActiveSupport::TestCase
   end
 
   test '存在しない日報のID番号のURLを言及しても言及は作成されない' do
-    @report.content = "http://localhost:3000/reports/#{- @report.id}は存在しない日報だ。"
+    @report.content = "http://localhost:3000/reports/#{Report.last.id + 1}は存在しない日報だ。"
```

Comment:
```suggestion
    @report.content = "http://localhost:3000/reports/#{Report.maximum(:id) + 1}は存在しない日報だ。"
```

こうした方がより意図が明確になると思います


https://api.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-maximum

Reply:
コメントありがとうございました！
指摘いただいた点を修正してマージします。
この度はレビューありがとうございました🙇🏻‍♀️